img/c/culturedcode.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="234.365" height="279.361"><path d="M16.94 38.4h202v219h-202z" fill="#fff"/><g fill-rule="evenodd"><path d="M234.35 157.88v7.25c0 .45-.09.9-.15 1.38l-6.07 77h-.05a51 51 0 0 1-1.21 8.33c-1.59 6.65-3.31 11-9.68 16.87-7.88 7.27-19.54 9.73-26.4 10.24-5.91.45-48.91.41-48.91.41H92.7s-43 0-48.9-.41c-6.85-.51-18.52-3-26.4-10.24-6.36-5.86-8.08-10.22-9.67-16.87a55.46 55.46 0 0 1-1.31-11.04L.04 165.32v-1.75a.49.49 0 0 0 0-.12v-7.43l.05-15.84.52-22.65 1.35-46.92 1.43-38.13a33.19 33.19 0 0 1 11-21.75C21.18 4.89 32.31 1.16 41.24.6 49.81.07 62.14 0 68.47 0h97.23c6.33 0 18.65.07 27.22.6 8.94.56 20.07 4.29 26.85 10.13a33.21 33.21 0 0 1 11 21.75c.51 8 1.15 29.34 1.43 38.13l1.34 46.92.53 22.65c.15 3.74.35 11.16.28 17.7zM52.09 181.73c3.42.28 11.78 1.19 17.1 3.75 5.05 2.43 7.72 5.69 8.55 8.1 1.28 3.69 1.33 21.52 1.5 25.43.13 2.94.32 13 .45 15.67.38 7.72.32 8.93 3.83 11.18 3.91 2.51 9.82 2.63 11.55 2.7s17.85 0 17.85 0h8.62l19.35-.08c2.91-.16 7.88-1.27 9.9-2.55a7.45 7.45 0 0 0 3.53-4.35c.28-1.41.83-24.2 1.05-30.07l.75-15.9a10.43 10.43 0 0 1 3.07-5.55 24.22 24.22 0 0 1 9.38-5.48 77.47 77.47 0 0 1 14.17-2.7c3.12-.28 8.64-.67 11.93-.67 2.91 0 9.89-2 13.48-4.93l-2.61-121.45c-.12-4.91-4.81-8.28-9.67-8.28l-157.35-.08c-4.84 0-9.63 3.15-9.75 8.29L26.18 176.3c3.6 2.94 10.56 5 13.46 5 1.32-.02 7.92.06 12.45.43z" fill="#2274e5"/><path d="M150.79 245.93c-2 1.28-7 2.4-9.9 2.55s-19.35.08-19.35.08H95.07c-1.73-.07-7.64-.19-11.55-2.7-3.51-2.25-3.45-3.46-3.83-11.18-.13-2.63-.32-12.73-.45-15.67l-.5-16.58h77l-.37 9.08-1.05 30.07a7.45 7.45 0 0 1-3.53 4.35z" fill="#bec2c8"/></g><path d="M117.36 161.16c-.74 0-2.39-1.3-3.21-2.09s-9.45-8.95-17.85-16.67l-19.83-18.33c-1.1-1.08-.75-1.47-.51-1.71.83-.88 9.3-8.47 10-8.92.19-.14.64 0 1.18.45.06 0 5.42 4.1 11.14 8.54l13.8 10.94s3.62-4.79 31.48-30.12c18.78-17.07 29.33-25.41 30.61-26.26a2.18 2.18 0 0 1 1-.38.73.73 0 0 1 .58.33s-.26-.3 0 0 5.25 6.75 5.62 7.43c.34.6.28.65 0 .92a426.22 426.22 0 0 0-30 32.92c-21 25.72-30.51 39.45-31.53 40.8 0 0 1.08-1.49 0 0s-1.85 2.15-2.48 2.15z" fill="#2c3b4a"/></svg>
```

Comment:
There's always room for improvement :smile: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 234.4 279.4"><path d="M77 202.5h80V251H77z" fill="#bec2c8"/><path d="M25 45h185v160H25z" fill="#fff"/><path d="M117.4 161.2c-.8 0-2.4-1.3-3.3-2.1l-17.8-16.7-19.8-18.3c-1.1-1.1-.8-1.5-.5-1.7.8-1 9.3-8.5 10-9a282.4 282.4 0 0 1 12.3 9l13.8 11s3.6-4.8 31.5-30.2c18.7-17 29.3-25.4 30.6-26.2a2.2 2.2 0 0 1 1-.4.7.7 0 0 1 .6.3s-.3-.3 0 0l5.6 7.5c.3.6.3.6 0 .9a426.2 426.2 0 0 0-30 33c-21 25.6-30.5 39.4-31.6 40.7 0 0 1.1-1.5 0 0s-1.8 2.2-2.4 2.2z" fill="#2c3b4a"/><path fill="#2274e5" d="M234.3 157.9v7.2l-.1 1.4-6 77h-.1a51 51 0 0 1-1.2 8.3 29 29 0 0 1-9.7 17 45.8 45.8 0 0 1-26.4 10.1c-6 .5-49 .5-49 .5h-49s-43 0-48.9-.4a46 46 0 0 1-26.4-10.3 29 29 0 0 1-9.7-16.9 55.5 55.5 0 0 1-1.3-11L0 165.3v-1.7a.5.5 0 0 0 0-.2v-23.1l.6-22.7 1.4-47 1.4-38.1a33.2 33.2 0 0 1 11-21.8A51 51 0 0 1 41.2.6c8.6-.5 21-.6 27.3-.6h97.2c6.3 0 18.6 0 27.2.6 9 .6 20 4.3 26.9 10.1a33.2 33.2 0 0 1 11 21.8c.5 8 1.1 29.3 1.4 38.1l1.3 47 .6 22.6c.1 3.7.3 11.1.2 17.7zM52.2 181.7a56 56 0 0 1 17 3.8c5.1 2.4 7.8 5.7 8.6 8 1.3 3.8 1.4 21.6 1.5 25.5.2 3 .4 13 .5 15.7.4 7.7.3 9 3.8 11.2 4 2.5 9.8 2.6 11.6 2.7h26.4l19.4-.1c2.9-.2 7.9-1.3 9.9-2.6a7.5 7.5 0 0 0 3.5-4.3c.3-1.4.8-24.2 1-30l.8-16a10.4 10.4 0 0 1 3-5.5 24.2 24.2 0 0 1 9.5-5.5 77.5 77.5 0 0 1 14.1-2.7c3.2-.3 8.7-.7 12-.7 2.9 0 9.9-2 13.4-5l-2.6-121.4c0-4.9-4.8-8.2-9.6-8.2l-157.4-.1c-4.8 0-9.6 3.1-9.7 8.3l-2.6 121.5c3.6 3 10.5 5 13.4 5 1.4 0 8 0 12.5.4z"/></svg>
```

Reply:
@Carlgo11 Thank you! I've committed the change. I see the difference between the two files now - my original had whitespace to the right and below and yours is cropped like the one from the press kit (which is what I started with). I'm not sure how that extra whitespace was introduced. Possibly from running it through `svgo`.